Feat #5968/forgot password (#5971)
vuestorefront/vue-storefront: packages/core/docs/guide/authentication.md

  }
}
</script>
```

## Forgot Password

> Usually the process of resetting user password after forgetting consists of two steps:

Reviewer: ```suggestion
Usually, the process of resetting a user password consists of two steps:
```

> 1. Generate reset password token with an email address.

Reviewer: ```suggestion
1. Generating reset password token for a given email address:
```

```vue
<template>

>   <form @submit.prevent="reset({ email: form.value.email })">

Reviewer: `form` object doesn't exist in `setup` method

Author: Corrected

    <!-- form fields -->
    <button type="submit" :disabled="loading">Reset Password</button>
  </form>
</template>

<script>
import { useForgotPassword } from '{INTEGRATION}';

export default {
  setup () {
    const { reset, loading } = useForgotPassword();

    return {
      reset,
      loading
    }
  }
}
</script>
```

> 2. Set new user password using the token and new password.

Reviewer: ```suggestion
2. Setting a new user password using the token and new password.
```

```vue
<template>

>   <form @submit.prevent="change({ tokenValue: <TOKEN_FROM_URL>, newPassword: form.value.password })">

Reviewer: `form` object doesn't exist in `setup` method
